### PR TITLE
chore(Scripts/ShatteredHalls): Rework Nethekurse

### PR DESCRIPTION
**Pull request by Nyeriah**
## Changes Proposed:
-  Fix the intro roleplay, nethekurse always walks, fix void zones 
-  

## Issues Addressed:

- Updates https://github.com/azerothcore/azerothcore-wotlk/issues/16070

## Tests Performed:

- Tested ingame
- 

## How to Test the Changes:

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Gultask — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1684108506441826000.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+(@NPC, 4, 178.51125, 287.97794, -8.183065, 0);
+
+UPDATE `creature_template` SET `MovementType` = 2 WHERE `entry` IN (16807, 20568);
+
+DELETE FROM `creature_template_addon` WHERE `entry` IN (16807, 20568);
+INSERT INTO `creature_template_addon` (`entry`, `path_id`) VALUES
+(16807, @NPC),
+(20568, @NPC);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 22) AND (`SourceGroup` = 3) AND (`SourceEntry` = 17083) AND (`SourceId` = 0);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1684108506441826000.sql:17]: 🥺 
```suggestion
(20568, @NPC);

DELETE FROM `smart_scripts` WHERE (`entryorguid` = 17083) AND (`source_type` = 0) AND (`id` IN (1, 2, 3));
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(17083, 0, 2, 0, 4, 0, 100, 512, 0, 0, 0, 0, 0, 45, 1, 1, 0, 0, 0, 0, 19, 16807, 50, 0, 0, 0, 0, 0, 0, 'Fel Orc Convert - On Aggro - Set Data 1 1'),
(17083, 0, 3, 0, 6, 0, 100, 512, 0, 0, 0, 0, 0, 45, 1, 2, 0, 0, 0, 0, 19, 16807, 50, 0, 0, 0, 0, 0, 0, 'Fel Orc Convert - On Just Died - Set Data 1 2');

DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 22) AND (`SourceGroup` = 3) AND (`SourceEntry` = 17083) AND (`SourceId` = 0);
```

**Comment by elthehablo** [comment]
Is this working now? I tried testing but couldn't get the movement to trigger last time.

**Comment by elthehablo** [comment]
The pause in his path to applaud when a peon dies is not long enough so he does not do the emote, only face the group for a bit. But this is merged now, so might as well submit an issue

**Comment by Nyeriah** [comment]
I left the issue open for a reason, this is not finished and might still need to be checked. There were targeting issues that I was not able to address

**Pull request merged**